packages/plugin-react-refresh/index.js

```diff
@@ -1,6 +1,7 @@
 // @ts-check
 const fs = require('fs')
 const { transformSync, ParserOptions } = require('@babel/core')
+const { isArray } = require('util')
```

Comment:
If I see it correct, you don't use this below, cause you use `Array.isArray`
Was this expected?